_seattle/API Chaya.md

```diff
@@ -1,18 +1,18 @@
 ---
 name: API Chaya
-address: API Chaya, P.O. Box 14047, Seattle, WA 98114
+address: 
 lat: 
```

Comment:
I did go through and manually put the lat/long values in for all the resources that have addresses, but APY Chaya mistakenly had a mailing address in the address field (it actually doesn't have an address). All of the other resources with addresses should have lat/longs though.